test/e2e/support/utils.js

```diff
@@ -1,24 +1,31 @@
 /**
  * Node dependencies
  */
-import path from 'path';
-import url from 'url';
+import { join } from 'path';
+import { URL } from 'url';
 
-const BASE_URL = 'http://localhost:8888';
-const USERNAME = 'admin';
-const PASSWORD = 'password';
+const WP_BASE_URL = process.env.WP_BASE_URL || 'http://localhost:8888';
+const WP_USERNAME = process.env.WP_USERNAME || 'admin';
+const WP_PASSWORD = process.env.WP_PASSWORD || 'password';
```

Comment:
It would be awesome to add a `DEBUG` env variable that automatically triggers `{ headless: false, slowMo: 100 }`

Reply:
Yeah, or a custom slowMo... If I don't set a slowMo (with headless), I get segmentation fault 11.

Reply:
Done.